cmd/release-controller/http_helper.go

```diff
@@ -16,22 +16,22 @@ import (
 )
 
 type ReleasePage struct {
-	BaseURL string
-	Streams []ReleaseStream
+	BaseURL string          `json:"baseURL"`
```

Comment:
I don't think we want these to be public APIs.

If the use case is "you want to get the latest release for a given stream", then the URL would be

```
/releasestream/{name}/latestaccepted
```

and the implementation should create a new object that has fields `name`, `pullSpec` and that's it.

Reply:
Makes sense, I’ll reduce it to those fields and change the route